frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -477,7 +485,17 @@ function aggregateIdLabels(items: { [id: string]: string }[]): {
   return items.reduce((memo, item) => ({ ...memo, ...item }), {});
 }
 
-function useWMGQueryRequestBody() {
+const EMPTY_FILTERS = {
+  datasets: undefined,
+  developmentStages: undefined,
+  diseases: undefined,
+  ethnicities: undefined,
+  sexes: undefined,
+};
```

Comment:
Is there a type we can assign to this object to make sure it doesn't break if we add any filters in the future?

Reply:
Ooh yes, really good point. Thank you! Done!